samples/exposed-migration/gradle.properties

```diff
@@ -0,0 +1,3 @@
+exposedVersion=0.52.0
```

Comment:
Inside the root `gradle.properties` there is `version` parameter. Could it be used instead?

Reply:
I don't think so, because the sample is supposed to be a stand-alone project that's not dependent on another project.

Reply:
I would keep it as it is here cause it's easier for users to understand